cmd/scale.go

```diff
@@ -320,6 +345,15 @@ func (sc *scaleCmd) run(cmd *cobra.Command, args []string) error {
 	addValue(parametersJSON, sc.agentPool.Name+"Count", countForTemplate)
 
 	switch orchestratorInfo.OrchestratorType {
+	case api.OpenShift:
+		err = transformer.NormalizeForOpenShiftVMASScalingUp(sc.logger, sc.agentPool.Name, templateJSON)
+		if err != nil {
+			log.Fatalf("error tranforming the template for scaling template %s: %s", sc.apiModelPath, err.Error())
+			os.Exit(1)
```

Comment:
Fatal already os.Exits

Reply:
the whole file needs a cleanup - I'm a bit torn between A) matching existing code, B) adding cleaner code, C) trying to improve the whole thing.  I'll move my stance from A) to B), how about that?